### ISSUE TITLE
Residual function doesn't work with binomial factor response

### ISSUE THREAD

**Issue by florianhartig** [glmmTMB/glmmTMB#314]
Example:

```
library(DHARMa)
library(glmmTMB)

testData = createData(sampleSize = 200, family = binomial(), factorResponse = T)
fittedModel <- glmmTMB(observedResponse ~ Environment1 + (1|group) , family = "binomial", data = testData)
residuals(fittedModel)
```

**Comment by florianhartig** [comment]
still not included in 0.2.1 on CRAN

**Comment by bbolker** [comment]
oops. Thanks for the heads-up. This got added to the 'profile' branch, not master.  My fault.  PR now pending ...
